pandas/core/series.py

```diff
@@ -1243,6 +1245,16 @@ def quantile(self, q=0.5):
         ----------
         q : float or array-like, default 0.5 (50% quantile)
             0 <= q <= 1, the quantile(s) to compute
+        interpolation : {'linear', 'lower', 'higher', 'midpoint', 'nearest'}
```

Comment:
@jorisvandenbossche @jreback 
While trying to make a `_shared_doc` in `generic.py`, there are more replacements than common text. The only common text is the  Parameters `q` and `interpolation` descriptions. 
Is it better I make ` _shared_docs['quantileparamscommon']` in `generic.py` and do a `@Substitution` on `Dataframe.quantile` and `Series.quantile` docstrings ?